packages/core/parcel/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "parcel",
+  "version": "2.0.0",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/parcel-bundler/parcel.git"
+  },
+  "bin": {
+    "parcel2": "src/bin.js"
```

Comment:
Just for now, until we release. That way you can have both Parcel 1 and 2 on your system at the same time.